packages/layout-grid/Row.js

```diff
@@ -0,0 +1,28 @@
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const Row = (props) => {
+  const {
+    children,
+    className,
+    ...otherProps
+  } = props;
+
+  const classes = classnames('mdc-layout-grid__inner', className);
+
+  return (
+    <div className={classes} {...otherProps}>{children}</div>
```

Comment:
we should also allow for devs to update tag name. See https://github.com/material-components/material-components-web-react/pull/327/files#diff-a4ae7491ba4f26fa221dde04643b6fa0R7